package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "el-msteams-post-action",
+  "version": "1.0.0",
+  "description": "Custom github action to post notifications to MS Teams",
+  "main": "lib/main.js",
```

Comment:
First of all sorry for the delay, for some reason some comments of yours do not show up in the main page, only if I go to the changed files.
The preparation of the action is done in two steps here :
- first compile the typescript code to javascript --> after an npm run build, the .js files will be updated in the ./lib directory
- then combine all .js code in one file for the action to trigger that --> this will be created under dist and will be referenced in the action.yml for the action itself.
I added some details in the README on how to develop further in the project